cmd/neofs-node/config.go

```diff
@@ -306,9 +306,11 @@ type internals struct {
 
 	log *logger.Logger
 
-	wg      *sync.WaitGroup
-	workers []worker
-	closers []func()
+	wg           *sync.WaitGroup
+	workers      []*worker
+	namedWorkers map[string]*worker
```

Comment:
What about storing name in a `worker` and sticking with the old approach here? To me map is an overkill here.

Reply:
In this case removing from slice will be a bit complex. I see that number of workers is too low to think about this as about time consuming task. Working on implementing of the same approach as was done for logger.

Reply:
Refactored, please review.